### DISCUSSION TITLE
Does this work with the Spaghetti Detective plugin?

### DISCUSSION THREAD

**Discussion by SirDragonClaw** [feelfreelinux/octo4a#64] [Q&A]
I know the Spaghetti Detective plugin can be very picky about webcams, just wondering if anyone has tried it and got it working? I imagine it would work even better than normal (and its normally very good) with the great cameras on most phones.

**Comment by feelfreelinux** [comment]
As far as I remember, people have reported success with Spaghetti Detective. In case you run into any problem - please let me know, I'll take care of it :)

**Reply by dragontype191** [reply]
Hello, as a user of spaghetti detective, I can definitely say it works... Kinda. It isn't compatible with their pro accounts streaming, so you're limited to a .1 fps stream (yes, one frame every 10 seconds). I contacted spaghetti detective support on this and they said that spaghetti detective is only supported on raspberry pi devices. Would love to see some kind of workaround to get the 25fps steaming like I get on a pi.

**Reply by kelcyknapp** [reply]
This! I'm begging =(

**Reply by midnightsmith89** [reply]
Works fine for me with 25fps streaming, using an old Pixel 2 XL

**Reply by kelcyknapp** [reply]
Really? No special settings or anything? Because mine is very much set at .1 fps, and I have 1800 hours of pro plan ;-;

**Reply by midnightsmith89** [reply]
Nope, just loaded and went! I feel like it's very phone specific, just like the OTG charge and sync issue

**Reply by Etienneg93** [reply]
@midnightsmith89 I have an old Pixel 2 XL as well that I set up using the spaghetti detective, signed up for the pro sub, restarted the server made sure that I did not have compatability mode enabled but still can't get the 25 fps version to work.

**Reply by ninjacarlo** [reply]
I have a P2XL too, I'll soon try it out, hope it works, then I'll report here.
Is there any way to train the AI for yourself using spaghetti's github therefore getting unlimited detective hours for free?

**Reply by BrandonTwede** [reply]
Are y'all still running octo4a and obico together? I am having webcam issues and cannot get it to work.
